### PR TITLE
Cleanup news

### PR DESCRIPTION
**Pull request by lucasvanhalst**
See 
- https://github.com/our-city-app/mobicage-android-client/pull/208
- https://github.com/our-city-app/cordova-mobicage-plugin/pull/8

# TODO
- TBD block registration of apps older then x
- TBD block registration of apps that use the old news

### CODE REVIEW

File: src-test/rogerthat_tests/mobicage/models/test_news.py [hunk 1]
```diff
@@ -1,87 +0,0 @@
...
-import mc_unittest
-
-from rogerthat.rpc import users
-from rogerthat.models import UserProfile
-from rogerthat.models.news import NewsItem
-from rogerthat.utils import get_epoch_from_datetime, now
-
-class TestNewsItem(mc_unittest.TestCase):
-
-    def test_news_item_target_audience(self):
```
Comment (RubenMattan) [inline src-test/rogerthat_tests/mobicage/models/test_news.py:30]: keep this

File: src-test/rogerthat_tests/mobicage/service/test_news.py [hunk 2]
```diff
@@ -192,50 +186,12 @@ def test_properties(self):
         original_statistics = getattr(news_item.statistics[app_id], statistic_property, default_stats)
         setattr(news_item.statistics[app_id], statistic_property, original_statistics)
 
-    def test_roles_matching(self):
```
Comment (RubenMattan) [inline src-test/rogerthat_tests/mobicage/service/test_news.py:195]: keep this

File: src/rogerthat/api/news.py [hunk 3]
```diff
@@ -19,33 +19,14 @@
...
     SaveNewsGroupServicesRequestTO, SaveNewsGroupFiltersResponseTO, SaveNewsGroupFiltersRequestTO, \
     GetNewsGroupResponseTO, GetNewsGroupRequestTO, SaveNewsStatisticsResponseTO, SaveNewsStatisticsRequestTO
 from rogerthat.utils.app import get_app_id_from_app_user
 
 
-@expose(('api',))
-@returns(GetNewsResponseTO)
-@arguments(request=GetNewsRequestTO)
-def getNews(request):
-    from rogerthat.bizz.news import get_news_for_user
```
Comment (RubenMattan) [inline src/rogerthat/api/news.py:34]: we should first cleanup all apps that use this

**Comment by lucasvanhalst** [comment]
logs that show usage of getNews: https://console.cloud.google.com/logs/query;query=%22%5C%22f%5C%22:%5C%22com.mobicage.api.news.getNews%5C%22,%22%0AprotoPayload.resource%3D%22%2Fjson-rpc%22;timeRange=P7D?project=rogerthat-server

so it's mostly the ios version of threefold

**Pull request merged**